### PR TITLE
Add resource actions commands

### PR DESCRIPTION
**Pull request by owenthereal**
# Description of change

```
$ m resource validate tunnel
Resource "tunnel" validation is successfully triggered!
Meroxa will try to connect to the resource for 60 minutes and send an email confirmation after a successful resource validation.

$ m resource rotate-tunnel-key tunnel
Rotating tunnel key will restart the tunnel and disconnect existing connections.
To proceed, type "tunnel" or re-run this command with --force
▸ tunnel
Resource "tunnel" tunnel key is successfully rotated!
Paste the following public key on your host:
ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQDMmFyvz3uE/ieininL3510uYaw8+lfTFzevgX4ASSsJ7J0mP2L9ozuPAxA8pzqqc8sWfIo4uUPdDKYyH2lPM8kcvbZ1G43WQb0D8srcL3FWWWRka0/2nOADnGVqlbq4DHPW4KbyURvQw7bVXqrNQ9un/TvSEM8yMTJirxVochfqgQUO2MVlVYFVMBJyfIWWjanQ570QiiPOzVjCQn4y1UWx3Gy6mOTwAqw0nbGsEJIm65dy/SRXhqx+7R+3y1rPLYQ793uSzcaEksnne627izd10MXJo39L/aeVJfyCniDQJMkqVDzhVPMKmN5dUksZt5o2qN2G9579ZB4vrgMxpP7xQ/MJTjF5+x9D4jFW7cLI1Ydim2RGlxfQ8+TGqKJQ3HyVmrvfNFa4GyV4BC7J5Z9cHcEIe5NqDUVR4C62mIzYl7rCJ8p63LZ/L5ClitgjJpSE5jLVLEH6t3jv5j9TxfMzLF+4f+sKB5689efVcOWtNiNcXa+l3Xheyewzy8eeg1M2VG6QvqqZC/o89lwBRAZdhHTx40qMsWcsXogC3dp/GtNVqWc9dupGNAMZ6wsU9swhJItzY/QtSaQnyCXJeKAth2xjE45pjXQ525OTsncMBhgBC1we0kKf84/nsN/80QJkABNRuHvMA0xDycBqwxt3y5p8tNLlye6gv01AlADwQ==
Meroxa will try to connect to the resource for 60 minutes and send an email confirmation after a successful resource validation.
```

Fixes https://meroxa.atlassian.net/browse/PLATFORM-597 & https://meroxa.atlassian.net/browse/PLATFORM-598

It depends on https://github.com/meroxa/meroxa-go/pull/42.

# Type of change

- [x]  New feature
- [ ]  Bug fix
- [ ]  Refactor
- [ ]  Documentation

# How was this tested?

- [x]  Unit Tests
- [ ]  Tested in staging

# Demo

See above.

### CODE REVIEW

File: cmd/meroxa/root/resources/rotate_tunnel_key.go [hunk 1]
```diff
@@ -54,6 +54,11 @@ func (u *RotateTunnelKey) Docs() builder.Docs {
 	}
 }
 
+func (u *RotateTunnelKey) Confirm(ctx context.Context) string {
+	u.logger.Infof(ctx, "Rotating tunnel key will restart the tunnel and disconnect existing connections.")
```
Comment (jayjayjpg) [inline cmd/meroxa/root/resources/rotate_tunnel_key.go:58]: Nice one! 👍🏾

File: cmd/meroxa/root/resources/rotate_tunnel_key.go [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+func (u *RotateTunnelKey) ParseArgs(args []string) error {
+	if len(args) < 1 {
+		return errors.New("requires resource name")
+	}
+
+	u.args.Name = args[0]
+	return nil
+}
+
+var (
```
Comment (dianadoherty) [inline cmd/meroxa/root/resources/rotate_tunnel_key.go:100]: If I remember correctly, @raulb wants to keep these vars near the top of the file. Think: https://github.com/meroxa/cli/blob/ae8f69c52aa2fcd581f12562b74b65c1619f2c0e/cmd/meroxa/root/resources/list.go#L28

File: cmd/meroxa/root/resources/validate.go [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+func (u *Validate) ParseArgs(args []string) error {
+	if len(args) < 1 {
+		return errors.New("requires resource name")
+	}
+
+	u.args.Name = args[0]
+	return nil
+}
+
+var (
```
Comment (dianadoherty) [inline cmd/meroxa/root/resources/validate.go:95]: Same here for being top of file

**Review by dianadoherty — APPROVED** [review]
small nit

File: cmd/meroxa/root/resources/rotate_tunnel_key.go (hunk 2 above)
Comment (owenthereal) [reply cmd/meroxa/root/resources/rotate_tunnel_key.go]: Sure, i can do that.

**Pull request merged**